src/sentry/runner/decorators.py

```diff
@@ -25,3 +34,21 @@ def inner(ctx, *args, **kwargs):
             configure()
         return ctx.invoke(f, *args, **kwargs)
     return update_wrapper(inner, f)
+
+
+def log_level_option(f):
+    "Give ability to configure global logging level. Must be used before configuration."
+    import click
+    from functools import update_wrapper
+
+    @click.pass_context
+    @click.option('--loglevel', '-l', default=None,
```

Comment:
We should also set:

```
envvar='SENTRY_LOG_LEVEL'
```

so it can be set from the runner. `SENTRY_LOG_LEVEL=info sentry run web`